doc/api/webstreams.md

```diff
@@ -613,15 +613,24 @@ added: v16.5.0
   {ReadableStream} is closed or rejected if the stream errors or the reader's
   lock is released before the stream finishes closing.
 
-#### `readableStreamBYOBReader.read(view)`
+#### `readableStreamBYOBReader.read(view[, options])`
 
 <!-- YAML
 added: v16.5.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/54044
+    description: Added `min` option.
```

Comment:
Question: how do we handle the API history for backported features? 🤔 
* The linter told me to update the URL to point to *this* PR for the v20 docs, so I did that. I assume that's correct?
* Once this has shipped in a v20 release, do I need to open a separate PR on `main` to update the API history there for v20? Or is there some magic tooling that merges the API history across all active release branches?

Reply:
Yes, that's correct.

No need to update a pr for `main`. v20 API has been changed by this PR, `main`/`v22` API has been changed by https://github.com/nodejs/node/pull/50888.